test/RachisTests/DatabaseCluster/ShardedAtomicClusterReadWriteTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Raven.Client.Documents;
+using Raven.Client.ServerWide;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace RachisTests.DatabaseCluster
+{
+    public class ShardedAtomicClusterReadWriteTests : AtomicClusterReadWriteTestsBase
+    {
+        public ShardedAtomicClusterReadWriteTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected override DocumentStore GetDocumentStore(Options options = null, [CallerMemberName] string caller = null)
```

Comment:
remove, use .Sharding.GetDocumentStore
base method should have IDocumentStore as a parameter

Reply:
Actually they should use RavenData and RavenTheory

Reply:
I think this is not relevant anymore after @karmeli87 's commit but I changed the implementation of the overridden `GetDocumentStore` to use `Sharding.GetDocumentStore`